src/grpc/build.rs

```diff
@@ -40,6 +40,7 @@ pub struct ImageBuildFrontendOptions {
     pub(crate) shmsize: u64,
     pub(crate) secrets: HashMap<String, SecretSource>,
     pub(crate) ssh: bool,
+    pub(crate) named_contexts: HashMap<String, String>,
```

Comment:
I'm seeing the Go implementation uses a `map[string]NamedContext` where a `NamedContext` is a:

```go
type NamedContext struct {
	Path  string
	State *llb.State
}
```

We should probably make this implementation forward-compatible with any future ideas, even if we don't need to implement everything in one go...

Could we use a `NamedContext` struct with just the `path` field for the moment, instead of a String as the value of this `HashMap` ?

Reply:
Sure, makes sense. Should be resolved with https://github.com/fussybeaver/bollard/pull/467/commits/cbf3a81b9794a506a2da89550a73d49804ff7411.